napari/layers/surface/normals.py

```diff
@@ -44,9 +44,5 @@ class SurfaceNormals(EventedModel):
     Represents both face and vertex normals for a surface mesh.
     """
 
-    face: Normals = Field(
-        Normals(mode=NormalMode.FACE, color='orange'), allow_mutation=False
-    )
-    vertex: Normals = Field(
-        Normals(mode=NormalMode.FACE, color='blue'), allow_mutation=False
-    )
+    face: Normals = Field(Normals(mode=NormalMode.FACE, color='orange'))
+    vertex: Normals = Field(Normals(mode=NormalMode.FACE, color='blue'))
```

Comment:
I thought that wouldn't work because you would end up with the same object in different models, just liek when you use `[]` as default in a function, but I guess that' not a problem here because of default validation creating a new object?

Reply:
```python
In [3]: class N(EventedModel):
   ...:     x: int = 1
   ...:

In [4]: class M(EventedModel):
   ...:     n: N = N()
   ...:

In [5]: m1 = M()

In [6]: m2 = M()

In [7]: m1.n is m2.n
Out[7]: False
```

Reply:
In plain `pydantic.BaseModel` I understood that it will always make a copy of the default value when creating a new instance (unlike a default value of a function argument that is only bound to one instance), like you found with the example using `EventedModel` above.

I'm not sure of the exact mechanics of this copy or how deep it is, but I think we've been relying on that behavior without any obvious issues. I'm also not sure how using `Field` would help if there were issues - I assume we'd need to do use `default_factory` instead to actually build an entirely new instance.